agent/tools/dispatch_sprints.go

```
@@ -0,0 +1,269 @@
+
+var sprintPathRE = regexp.MustCompile(`^SPRINT-\d{3}\.md$`)
+
+// readDispatchPlan parses a JSONL file into validated entries. Returns an error
+// on the first malformed or invalid line so problems surface before any
+// expensive LLM calls fire.
+func readDispatchPlan(path string) ([]dispatchEntry, error) {
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, fmt.Errorf("open %s: %w", path, err)
+	}
+	defer f.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `f.Close` is not checked

(errcheck)